### PR TITLE
Avatar implementation

### PR DESCRIPTION
**Pull request by YeahImRose**

### CODE REVIEW

File: osu.Game/Overlays/Toolbar.cs [hunk 1]
```diff
@@ -7,11 +7,11 @@
 using osu.Framework;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
 using osu.Framework.Graphics.Transformations;
 using osu.Game.Configuration;
 using osu.Game.GameModes.Play;
 using osu.Game.Graphics;
-using osu.Framework.Graphics.Sprites;
```
Comment (autismpp) [inline osu.Game/Overlays/Toolbar.cs]: Why is this necessary? It was already below `.Containers`.

File: osu.Game/Overlays/Toolbar.cs [hunk 2]
```diff
@@ -85,15 +86,15 @@ public override void Load(BaseGame game)
...
                     RelativeSizeAxes = Axes.Y,
-                    Children = new []
+                    Children = new Drawable[]
                     {
                         new ToolbarButton
                         {
                             Icon = FontAwesome.search
                         },
-                        new ToolbarButton
+                        new UserButton(5456671)
```
Comment (autismpp) [inline osu.Game/Overlays/Toolbar.cs]: Hardcoded userid..?

File: osu.Game/Overlays/Toolbar.cs [hunk 3]
```diff
@@ -85,15 +86,15 @@ public override void Load(BaseGame game)
...
                         {
-                            Icon = FontAwesome.user,
+                            Image = new Avatar(5456671, 48, 6),
```
Comment (autismpp) [inline osu.Game/Overlays/Toolbar.cs]: Same here.

**Comment by Lunar-Dust** [comment]
Closing to make a proper PR
